ui/js/controllers/perf/graphs.js

```diff
@@ -514,7 +524,9 @@ perf.controller('GraphsCtrl', [
                                          resultSetData: _.pluck(
                                              response.data[series.signature],
                                              'result_set_id'),
-                                         thSeries: jQuery.extend({}, series)
+                                         thSeries: jQuery.extend({}, series),
+                                         jobID: _.pluck(response.data[series.signature],
```

Comment:
jobIdData would probably be a better name. jobID implies that the variable refers to a singular value, rather than an array.